src/brogue/Monsters.c

```diff
@@ -1246,7 +1246,7 @@ static unsigned long successorTerrainFlags(enum tileType tile, enum subseqDFType
     }
 }
 
-unsigned long burnedTerrainFlagsAtLoc(pos loc) {
+unsigned long burnedterrainFlagsLoc(pos loc) {
```

Comment:
The casing change here (and in related functions) seems like a regression

Reply:
Whoops! Find+Replace regex gone amok. I re-read through the changes and thankfully this was harmless, though obviously unintentional. They should be all fixed now.